client/src/main/java/org/apache/rocketmq/client/ClientConfig.java

```diff
@@ -42,6 +42,7 @@ public class ClientConfig {
     protected String namespace;
     private boolean namespaceInitialized = false;
     protected AccessChannel accessChannel = AccessChannel.LOCAL;
+    private boolean asySendBlockMode = true;
```

Comment:
It is better to put this parameter in `DefaultMQProducer`

Reply:
thx, I agree with your suggestion. I will modify it.